app/core/Tracker2/BaseTracker.js

```diff
@@ -17,6 +36,8 @@ export default class BaseTracker {
     this.initialized = false
 
     this.setupInitialization()
+
+    this.loggineEnabled = (new URLSearchParams(window.location.search)).has(TRACKER_LOGGING_ENABLED_QUERY_PARAM);
```

Comment:
Looks like this has ~95% usage. [[Ref](https://caniuse.com/#feat=urlsearchparams)]
Is there worth being graceful degradation of this feature if the browser is missing it? Maybe wrapping it in a `try` and `catch` and defaulting the loggineEnabled to false.
Also the naming of `loggineEnabled` is a little confusing. Unsure it typo or if I am misreading.

Reply:
Yup updated